src/routes/RemediationDetails.js

```diff
@@ -131,70 +133,39 @@ class RemediationDetails extends Component {
-                                <GridItem>
-                                    <Card className='ins-c-card__plan-details'>
-                                        <CardHeader className='ins-m-card__header-bold'>
-                                            Playbook details
-                                        </CardHeader>
-                                        <CardBody>
-                                            <Stack>
-                                                <StackItem>Created by: { formatUser(remediation.created_by) }</StackItem>
-                                                <StackItem>Created: { formatDate(remediation.created_at) }</StackItem>
-                                                <StackItem>Last modified by: { formatUser(remediation.updated_by) }</StackItem>
-                                                <StackItem>Last modified: { formatDate(remediation.updated_at) }</StackItem>
-                                                {
-                                                    isBeta &&
-                                                    <StackItem className='ins-m-border-top'> Shared with: unknown </StackItem>
-                                                }
-                                            </Stack>
-                                        </CardBody>
-                                    </Card>
-                                </GridItem>
-                            </Grid>
+                            <Card>
+                                <CardHeader className='ins-m-card__header-bold'>Playbook Summary</CardHeader>
+                                <CardBody>
+                                    <Flex className='ins-c-playbookSummary' breakpointMods={ [{ modifier: FlexModifiers.column }] }>
+                                        <Flex className='ins-c-playbookSummary__overview'>
+                                            <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                                                <DescriptionList
+                                                    isBold
+                                                    title='Total systems'>
+                                                    { pluralize(totalSystems, 'system') }
+                                                </DescriptionList>
+                                            </FlexItem>
+                                        </Flex>
+                                        <DescriptionList className='ins-c-playbookSummary__settings' title='Playbook settings'>
+                                            <Flex>
+                                                <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                                                    Autoreboot:&nbsp;
+                                                    <b> { remediation.auto_reboot && remediation.needs_reboot ? 'Enabled' : 'Disabled' } </b>
```

Comment:
In the mocks this is dark green then "Enabled". It is not clear what color the disabled option should be. Also, when the Playbook does not require reboot at all we should probably call it out explicitly rather than say "Disabled". Let me get that clarified with UXD

Reply:
Sounds good! I'll leave this conversation open, but I'm going to push up a change for the "require(s)"